run-tests.py

```diff
@@ -13,6 +13,8 @@
 
 # These paths should all pass PEP-8 checks
 paths_to_check = [
+    'server/pulp/plugins/file/',
```

Comment:
I'm surprised at this. On my dev box, I do not see these particular errors, but I did find that file to have these problems. I wonder why my own machine isn't finding these problems?

Anyways, I fixed it.